modules/firelens_log_config/variables.tf

```diff
@@ -0,0 +1,37 @@
+variable "environment_name" {
+  type = string
+}
+
+variable "app_name" {
+  type = string
+}
+
+variable "task_name" {
+  type = string
+}
+
+variable "datadog_image" {
+  type = string
+}
+
+variable "dd_source" {
+  type    = string
+  default = "elixir"
```

Comment:
dd_source for datadog container is gonna be `datadog` so I think we might have to make that default and set the dd_source var in the other modules parameters.

Reply:
I'm a little confused..the PR you pointed me to for this has it as elixir ([PR](https://github.com/smartrent/identity/commit/30e9d928996472097ed3e67c7fa3aa11dfb03664))
<img width="841" alt="Screen Shot 2022-04-28 at 6 05 51 PM" src="https://user-images.githubusercontent.com/21345512/165861759-7dcdf83d-837e-4a76-9d82-26cbe0cbba6d.png">

But even at that, with it defaulting to elixir, we could still set a variable in the app modules and use that variable to set this one in the module reference...right? 💭

Reply:
Would you have any issues with having a `log_config` that sets elixir as the `dd_source` and a `datadog_log_config` that sets datadog as the `dd_source`?

Reply:
I separated them into individual log configs as seen [here](https://github.com/smartrent/nerves-hub-terraform/pull/30/files#diff-f275be9fd58c428c299725380fb9b144a187f2f678014da65f86108b50da97e8R142) and [here](https://github.com/smartrent/nerves-hub-terraform/pull/30/files#diff-f275be9fd58c428c299725380fb9b144a187f2f678014da65f86108b50da97e8R170)